aspnetcore/security/key-vault-configuration.md

```diff
@@ -198,7 +198,7 @@ For information on using the provider with a managed identity and an Azure DevOp
 
 ## Configuration options
 
-<xref:Microsoft.Extensions.Configuration.AzureKeyVaultConfigurationExtensions.AddAzureKeyVault*> can accept an <xref:Microsoft.Extensions.Configuration.AzureKeyVault.AzureKeyVaultConfigurationOptions>:
+AddAzureKeyVault can accept an AzureKeyVaultConfigurationOptions:
 
 ```csharp
 config.AddAzureKeyVault(
```

Comment:
This overload takes a client as the first parameter. https://github.com/Azure/azure-sdk-for-net/blob/e51946a4a7fcff55515ecec58c30e30a9ac423ff/sdk/extensions/Azure.Extensions.AspNetCore.Configuration.Secrets/api/Azure.Extensions.AspNetCore.Configuration.Secrets.netstandard2.0.cs#L23